catt/cli.py

```diff
@@ -103,28 +108,109 @@ def write_config(settings):
         raise CattCliError("No device specified.")
 
 
+def hunt_subtitle(video):
+    """"Searches for subtitles in the current folder"""
+    video_path = Path(video)
+    video_path_stem_lower = video_path.stem.lower()
+    for entry in os.listdir(video_path.parent):
+        entry_path = Path(entry)
+        if entry_path.is_dir():
+            continue
+        if entry_path.stem.lower().startswith(video_path_stem_lower) and \
+                entry_path.suffix.lower() in [".vtt", ".srt"]:
+            return entry
```

Comment:
This is not working, as you're not returning the absolute path. You could do this:
```for entry_path in video_path.parent.iterdir()```
(and thus, no longer need ```os```)
And then let the function return ```str(entry_path.resolve())```